src/code/z_actor.c

```diff
@@ -2594,18 +2594,18 @@ void func_800B9D1C(Actor* actor) {
     }
 }
 
-void Actor_DrawAllSetup(PlayState* play) {
-    play->actorCtx.undrawnActorCount = 0;
+void Actor_ResetLensActors(PlayState* play) {
+    play->actorCtx.numLensActors = 0;
     play->actorCtx.lensActorsDrawn = false;
 }
 
-s32 Actor_RecordUndrawnActor(PlayState* play, Actor* actor) {
-    if (play->actorCtx.undrawnActorCount >= 32) {
+s32 Actor_RecordLensActor(PlayState* play, Actor* actor) {
```

Comment:
I'm fine with how it is, but perhaps `AddToLensActor`, kind of like how we already have `AddToCategory` which does something similar.

Reply:
I wasn't too satisfied with `Actor_RecordLensActor`, I do like `Actor_AddToLensActors` better